From 8f59fb469fc47b2ee64453407b7d4ce4d8f29f63 Mon Sep 17 00:00:00 2001
From: Daniel Kolesa <d.kolesa@samsung.com>
Date: Thu, 11 Jun 2020 02:18:50 +0200
Subject: [PATCH 5/7] eina: only enable EINA_LOG_BACKTRACE when backtrace API
 is present

This prevents build breakage on platforms that either don't have
backtrace() or don't have the appropriate library for it installed.
---
 src/lib/eina/eina_log.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git src/lib/eina/eina_log.c src/lib/eina/eina_log.c
index 7c66ee0b53..531faba093 100644
--- src/lib/eina/eina_log.c
+++ src/lib/eina/eina_log.c
@@ -42,7 +42,10 @@
 #endif
 
 #include "eina_debug_private.h"
+
+#ifdef HAVE_BACKTRACE
 #define EINA_LOG_BACKTRACE
+#endif
 
 #include "eina_config.h"
 #include "eina_private.h"
-- 
2.27.0

